tests/test_holiday_groups.py

```diff
@@ -129,3 +130,25 @@ def __init__(self, *args, **kwargs):
         test_holidays._add_visakha_bucha("Visakha Bucha")
         test_holidays._add_visakha_bucha("Visaka Bochea", KHMER_CALENDAR)
         self.assertEqual(0, len(test_holidays))
+
+
+class TestTibetanCalendarHolidays(TestCase):
+    def test_add_tibetan_calendar_holiday(self):
+        # Check for out-of-range dates.
+        class TestHolidays(HolidayBase, TibetanCalendarHolidays):
+            def __init__(self, *args, **kwargs):
+                TibetanCalendarHolidays.__init__(self)
+                super().__init__(*args, **kwargs)
+
+        test_holidays = TestHolidays()
+
+        test_holidays._populate(2025)
```

Comment:
This is for testing out-of-bound holidays
```suggestion
        test_holidays._populate(2100)
```